airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -315,7 +315,7 @@ def execute(self, context: Context):
             # the next() call which the RowIterator does not resemble to. Hence, ignore the arg-type error.
             # Row passed to _validate_records is a collection of values only, without column names.
             self._validate_records(next(iter(job.result()), []))  # type: ignore[arg-type]
-            self.log.info("Current state of job %s is %s", job.job_id, job.state)
+            self.log.debug("Current state of job %s is %s", job.job_id, job.state)
```

Comment:
One issue with this is: if something goes wrong in the Triggerer, you wouldn't see any messages for a while